### ISSUE TITLE
add DevTools cases into manual runs

### ISSUE THREAD

**Issue by kjozwiak** [brave/qa-resources#82]
As per @BrendanEich, we need to add DevTools into our manual passes to at least make sure that the baseline is working as expected.

**Comment by srirambv** [comment]
This is for b-l?

**Comment by kjozwiak** [comment]
> This is for b-l?

Correct, it doesn't have to be super detailed but need to make sure that the basics are working.
